src/components/editor/UpdateResourceModal.jsx

```diff
@@ -50,7 +50,7 @@ class UpdateResourceModal extends Component {
             Do you want to overwrite these resource templates?
           </Modal.Body>
           <Modal.Footer>
-            <Button onClick={() => {this.props.update(this.state.rts, this.state.group)}}>Yes, overwrite</Button>
+            <Button onClick={async () => {await this.props.update(this.state.rts, this.state.group)}}>Yes, overwrite</Button>
```

Comment:
assuming my similar suggestion above is correct, i think here you could instead do the slightly simpler:
```js
onClick={() => this.props.update(this.state.rts, this.state.group)}
```

just implicitly returning the promise returned by `update`, instead of `await`ing the promise and wrapping that blocking in a new promise that resolves once the wrapped promise is done.

Reply:
I found that I needed to make this change. Before I did, the effects of `this.props.update()` were non-deterministic.